src/include/pbs_internal.h

```diff
@@ -193,6 +193,13 @@ extern "C" {
 /* Default value of preempt_sort */
 #define PBS_PREEMPT_SORT_DEFAULT	"min_time_since_start"
 
+/* Structure to store each server instance details */
+typedef struct pbs_server_instance
+{
+	char name[PBS_MAXHOSTNAME];
```

Comment:
shouldn't this be `PBS_MAXSERVERNAME + 1`

Reply:
I agree, addressed it.